src/Core/Conventional.Tests/Conventional/Conventions/Cecil/CecilConventionSpecificationTests.cs

```diff
@@ -515,5 +515,63 @@ public void LibraryCodeShouldCallConfigureAwaitWhenAwaitingTasks_FailsWhenConfig
             result.IsSatisfied.Should().BeFalse();
             result.Failures.Single().Should().Be(expectedFailureMessage);
         }
+
+        # region MustNotUseGuidNewGuid
```

Comment:
**remark**: I've introduced a region here to make it obvious that all of this is associated with the `MustNotUseGuidNewGuid` convention. 

Might be worth splitting this file up.

Reply:
I'm not too worried about splitting these test files up, they tend to be easy enough to peruse. Can we remove the `#region` though? It conflicts with my core beliefs 😆